cli/deployment/utils.go

```diff
@@ -25,13 +30,87 @@ func unmarshal(data string) (map[string]interface{}, error) {
 // after import "absolutisation" deployment config might be copied to any location (as current CFT cli will copy its
 // copy to tmp folder.
 // Examples:
-//    /base/folder/config.YAML and ../script.py will concatenate to /base/script.py
-//    /base/folder/config.YAML and /base/folder/script.py will concatenate to /base/folder/script.py as long path already absolute
-func ReparentPath(parent string, child string) string {
-	// check if file already has absolute path
-	if child[0] == os.PathSeparator {
-		return child
+//    /base/folder/config.yaml and ../script.py will concatenate to /base/script.py
+//    /base/folder/config.yaml and /base/folder/script.py will concatenate to /base/folder/script.py as long path already absolute
+func ReparentPath(baseDir string, file string) string {
+	// check if import statement path already absolute
+	if file[0] == os.PathSeparator {
+		return file
 	}
-	dir := filepath.Dir(parent)
-	return filepath.Clean(filepath.Join(dir, child))
+	baseDir, _ = filepath.Abs(baseDir)
+	baseDir = filepath.Dir(baseDir)
+	relative := filepath.Clean(filepath.Join(baseDir, file))
+	result, err := filepath.Abs(relative)
+	if err != nil {
+		log.Fatalf("error creating absolute path, for file: %s, error: %v", relative, err)
+	}
+	return result
+}
+
+// check if string is yaml
+func IsYAML(text string) bool {
+	obj := struct{}{}
+	err := yaml.Unmarshal([]byte(text), obj)
+	return err == nil
+}
+
+// append string map B to map A, returns A
```

Comment:
Please follow doc comment convention as described at https://golang.org/doc/effective_go.html#commentary, in particular making sure this doc comment is a complete, grammatical sentence.

It is also worth noting in this comment that any keys in `b` that already exist in `a` are overwritten.